src/js/lib/monster.ui.js

```diff
@@ -3093,6 +3115,43 @@ define(function(require) {
 		}
 	};
 
+	/**
+	 * Get handlebars template to render an SVG icon
+	 * @param   {Object} args
+	 * @param   {String} args.id            Icon ID
+	 * @param   {String} [args.attributes]  Attributes to be added to the SVG tag
+	 * @return  {String}                    SVG icon template
+	 */
+	function getSvgIconTemplate(args) {
+		if (!_.isPlainObject(args)) {
+			throw TypeError('"args" is not a plain object');
+		}
+		if (!_.isString(args.id)) {
+			throw TypeError('"id" is not a string');
+		}
+		if (_.has(args, 'attributes') && !_.isPlainObject(args.attributes)) {
+			throw TypeError('"attributes" is not a plain object');
+		}
+		var iconId = args.id;
+		var iconPrefix = iconId.substring(0, iconId.indexOf('--'));
+		var attributes = _.get(args, 'attributes', {});
+		attributes.class
+			= _.chain(attributes)
```

Comment:
Please avoid linebreaks before or after `=` in an assignment.

Regarding lodash chaining, you can either keep `_.chain()` on the assignment line or just `_` and move `.chain()` on its own line.